src/tools/genpolicy/Cargo.toml

```diff
@@ -45,7 +45,6 @@ tokio = {version = "1.33.0", features = ["rt-multi-thread"]}
 oci = { path = "../../libs/oci" }
 
 # Kata Agent prototol.
-protocols = { path = "../../libs/protocols", features = ["with-serde"] }
 protobuf = "3.2.0"
```

Comment:
Shouldn't protobuf be removed too?

Reply:
Wasn't sure about this one. Removed. Thanks!